packages/pages/src/common/constants.ts

```diff
@@ -0,0 +1 @@
+export const TEMPLATES_ROOT = "src/templates";
```

Comment:
can this value be used in the other constants file so it's not defined in both places?

Reply:
This is outdated due to duval's suggestion